### PR TITLE
Add new CLI tool to clean dynamics values from validators db

### PR DESCRIPTION
**Pull request by vtleonardo**
This PR also fixes a lit bug found in the deploy.sh when passing the env variable to skip registration.

### CODE REVIEW

File: scripts/base-scripts/deploy.sh [hunk 1]
```diff
@@ -50,6 +50,9 @@ npx hardhat --network $NETWORK set-min-ethereum-blocks-per-snapshot --factory-ad
 cd $CURRENT_WD
 
 if [[ -n "${SKIP_REGISTRATION}" ]]; then
+    cd $BRIDGE_DIR
+    npx hardhat set-local-environment-interval-mining --network $NETWORK --interval 2500
+    cd $CURRENT_WD
```
Comment (diligentis) [inline scripts/base-scripts/deploy.sh:55]: This looks unrelated.

**Review by diligentis — CHANGES_REQUESTED** [review]
Remove unrelated code.

File: scripts/base-scripts/deploy.sh (hunk 1 above)
Comment (vtleonardo) [reply scripts/base-scripts/deploy.sh]: addressed!

**Review by diligentis — APPROVED** [review]
